src/ansys/tools/visualization_interface/backends/pyvista/pyvista.py

```diff
@@ -94,6 +108,14 @@ def __init__(
 
         self._use_trame = use_trame
         self._allow_picking = allow_picking
+        if self._allow_picking:
+            if allow_hovering:
+                logger.warning(
+                    "Picking and hovering are incompatible. Picking will take precedence."
+                )
+            self._allow_hovering = False
+        else:
+            self._allow_hovering = allow_hovering
```

Comment:
```
        if allow_picking and allow_hovering:
                logger.warning(
                    "Picking and hovering are incompatible. Picking will take precedence."
                )
                self._allow_hovering = False

        self._allow_hovering = allow_hovering
```

Reply:
I'm not a fan of ``if`` statements without an else. But open for discussion

Reply:
I had the very same conversation with AK a long time ago... he was about not "cluttering" the code unnecessary... I was opting more for readability. To me, up to Alex. This is just a cosmetic change, it can be disregarded :)

Reply:
I have no problem with an `if` without an `else`. The `else` is implied and is equivalent to having
```
else:
    pass
```
I think code readability is enhanced by leaving it off. I see no need to add these extra two lines.